meshtastic/interfaces/ble/notifications.py

```
@@ -228,3 +283,434 @@ def _get_callback(self, characteristic: str) -> Callable[[Any, Any], None] | Non
+                    Exception
+                ):  # noqa: BLE001 - notification callbacks must stay best effort
+                    _report_notification_error()
+                return
+            self.invoke_safe_execute_compat(
+                safe_execute,
+                _invoke_handler,
+                error_msg=error_msg,
+                fallback=_fallback_invoke_handler,
+            )
+
+        def _safe_legacy_handler(sender: Any, data: bytes | bytearray) -> None:
```

Ruff (0.15.6)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `sender`

(ANN401)